molecule/user_provided/converge.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Are some of these create/converge/... files literal C&P? If so, we should be able to use relative symlinks instead. We should only create the new files assuming something needs to be changed.

I think the structure of the default scenario should allow us to only provide a different vars.yml files.